packages/connect-thegraph-tokens/subgraph/src/aragon/mappings/MiniMeToken.ts

```diff
@@ -50,6 +49,8 @@ function _getMiniMeTokenEntity(previousBlock: BigInt, tokenAddress: Address): Mi
 
   let miniMeTokenEntity = MiniMeTokenEntity.load(miniMeTokenEntityId)
   if (!miniMeTokenEntity) {
+    aragon.processToken(tokenAddress, false)
```

Comment:
If the incoming token is an orphan, make sure that its registered in the AragonInfo registry.

Reply:
Might be useful to leave a comment here to explain this!